.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Report a bug
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Expected Behavior
+
+_Describe what you expected to happen._
+
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+1. Step 1
+2. Step 2
+...
+
+## Logs/Error Messages
+
+_Attach or paste appropriate logs and error messages to help us debug better._
+
+## Development Environment and Version
```

Comment:
@muthuswamyopn I'd like to combine all the version requests into one section.

Reply:
Modified @danfowler